src/load_balancer/host/load_balancer_host_factory.cxx

```diff
@@ -42,5 +42,35 @@ std::shared_ptr<LoadBalancer> LoadBalancerHostFactory::get_shared_instance(
 
 }
 
+std::shared_ptr<LoadBalancer> LoadBalancerHostFactory::get_shared_instance(
```

Comment:
It's not immedialte obvious to my why we need to have multiple implementations of the load balancer. We should just oppurtunistically (also) screen the protonic basis if its (optionally) present.

Reply:
Nevermind - I misread the code.